.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: Test
```

Comment:
Perhaps

```suggestion
name: Plugin Tester
```

Reply:
Mm personally I think Test is better (the action is run tests, `plugin tester` is a implementation detail of test